NEWS

```diff
@@ -651,6 +651,9 @@ CHANGES WITH 247 in spe:
           and are now understood by systemd-gpt-auto-generator, and the OS
           image dissection logic.
 
+        * IOWeight= has learnt to properly set the IO weight when using the
```

Comment:
247 has shipped, so this notice needs to be either moved at the top or removed

Reply:
The notice was removed, thanks.